infra/template/iam.tf

```diff
@@ -1,67 +1,53 @@
-# configurations for iam roles go here
-
 # ----------------------------------------------------------
 # 
 # IAM Roles for Users
 #
 # ----------------------------------------------------------
+# IAM roles/users created here:
+# wic-mt-github-actions: this is the user that will hold temporary credentials to AWS when the deploy action is running
+# wic-mt-deploy: gives the user permission to perform all ECR services (will be narrowed down later), and Update actions for ECS
 
-# resource "aws_iam_role" "wic_mt_dev" {
-  
+# resource "aws_iam_user" "github_actions" {
+#   name = "wic-mt-github-actions"
+# }
+# resource "aws_iam_role" "github_actions" {
+#   name = "deployment-action"
```

Comment:
should this be named `wic-mt-deploy` to match the comment above?

Reply:
no, the comment should be corrected. wic-mt-deploy is the policy that belongs to that role. I'll fix that